### PR TITLE
Add setIsFavorite funtion

### PR DESCRIPTION
**Pull request by PreslavKozovski**
With this function when "favorite" switch is pressed data item is changed. Previously switch had only animation and didn't change state of data item.

### CODE REVIEW

File: app/js/details.js [hunk 1]
```diff
@@ -35,6 +35,14 @@
             // Will use the Cordova InAppBrowser plugin when deployed to a device. Opens a new window in
             // the simulator
             window.open(currentBook.amazon_url, '_blank', 'location=yes');
+        },
+        setIsFavorite: function () {
+            Books.data.fetch(function () {
+                var dataItem = currentBook;
+                if(dataItem.is_favorite==true)dataItem.set("is_favorite", false);
```
Comment (bundyo) [inline app/js/details.js]: Please do leave more whitespace and add blocks, something like this:

``` javascript
if (dataItem.is_favorite === true) {
    dataItem.set("is_favorite", false);
} else {
    dataItem.set("is_favorite", true);
}
```

Alternatively you can write the whole thing like this:

``` javascript
dataItem.set("is_favorite", !dataItem.get("is_favorite"));
```

File: app/js/details.js [hunk 2]
```diff
@@ -35,6 +35,14 @@
...
+                else dataItem.set("is_favorite", true);
```
Comment (tsvetie) [inline app/js/details.js]: You can use 

dataItem.set("is_favorite", !dataItem.is_favorite);

File: app/js/details.js (hunk 1 above)
Comment (tsvetie) [inline app/js/details.js]: New line after variable definition.

**Comment by bundyo** [comment]
:+1: after comment

**Comment by tsvetie** [comment]
:+1:

**Pull request merged**